modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommercePaymentMethod.testcase

```diff
@@ -0,0 +1,388 @@
+		}
+	}
+
+	@description = "Verify an offline payment method can be added, edited, and deleted from system settings"
+	@priority = "3"
+	test AddEditAndDeleteOfflinePaymentMethod {
+		property portal.acceptance = "false";
+
+		task ("Given the System Integrator navigates to the Offline Payment Method Keys configuration under the Payment category in System Settings") {
+			SystemSettings.openSystemSettingsAdmin();
+
+			SystemSettings.gotoConfiguration(
+				configurationCategory = "Payment",
+				configurationName = "Offline Payment Method Keys",
+				configurationScope = "System Scope");
+		}
+
+		task ("When the the System Integrator clicks add") {
+			Button.clickAdd();
+		}
+
+		task ("And a key is provided for the offline payment method") {
+			Type(
+				locator1 = "SystemSettings#KEY",
+				value1 = "Test Offline Payment Method");
+		}
+
+		task ("And the save button is clicked") {
+			Button.clickSave();
+
+			Alert.viewSuccessMessage();
+		}
+
+		task ("Then the new entry is listed") {
+			AssertElementPresent(
+				key_configurationEntry = "Test Offline Payment Method",
+				locator1 = "SystemSettings#CONFIGURATION_ENTRY");
+		}
+
+		task ("When the the System Integrator edits the offline payment method and clicks update") {
+			SystemSettings.gotoConfigurationEntry(configurationEntry = "Test Offline Payment Method");
+
+			Type(
+				locator1 = "SystemSettings#KEY",
+				value1 = "Test Offline Payment Method 2");
+
+			Button.clickUpdate();
+		}
+
+		task ("Then the updated entry is listed") {
+			AssertElementPresent(
+				key_configurationEntry = "Test Offline Payment Method 2",
+				locator1 = "SystemSettings#CONFIGURATION_ENTRY");
+		}
+
+		task ("When the the System Integrator deletes the offline payment method") {
+			SystemSettings.deleteConfigurationEntry(configurationEntry = "Test Offline Payment Method 2");
+		}
+
+		task ("Then the entry should no longer be listed") {
+			AssertElementNotPresent(
+				key_configurationEntry = "Test Offline Payment Method 2",
+				locator1 = "SystemSettings#CONFIGURATION_ENTRY");
+		}
+	}
+
+	@description = "Verify an activated offline payment method can be used for an order."
+	@priority = "3"
+	test AnOfflinePaymentMethodCanBeUsedForAnOrder {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+
+			User.openUsersAdmin();
+
+			CommerceLogin.editPasswordCP(
+				newPassword = "test",
+				requirePasswordReset = "false",
+				userScreenName = "buyeruser");
```

Comment:
```suggestion

```
This editPasswordCP will not work since the password will be set as "test" automatically when calling `CommerceEntry.addAccountEntryUser()` macro.